contrib/forms/assessment_intake/view.php

```diff
@@ -56,14 +56,14 @@
 <img src="<?php echo $GLOBALS['images_static_relative'];?>/space.gif" width="572" height="1">
 <strong>Date:</strong>&nbsp; <?php print date('m/d/y'); ?><br /><br />
 <strong>SSN:</strong>&nbsp;<?php echo text($result['ss']);?><img src="<?php echo $GLOBALS['images_static_relative'];?>/space.gif" width="172" height="1">
-<strong>DCN:</strong>&nbsp;<input type="entry" name="dcn" value="<?php echo attr($obj["dcn"]);?>"><img src="<?php echo $GLOBALS['images_static_relative'];?>/space.gif" width="125" height="1">
-<label><strong>Location:</strong>&nbsp;<input type="entry" name="location" value="<?php echo attr($obj["location"]);?>"></label><br /><br />
```

Comment:
I guess this file needs a lot of refactoring.  I will create another PR to clean it.